File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        lastIndex = len(s)-1 
        numChar = 0
        while(lastIndex >= 0):
            if numChar > 0 and s[lastIndex].isspace():
                return numChar 
            
            numChar += 1 if s[lastIndex].isalpha() else 0
            lastIndex -= 1 
        return numChar

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int second = s.length()-1; 
        while(second >= 0 && Character.isSpace(s.charAt(second))) second--; 
        
        int first = second; 
        while(first >= 0 && Character.isLetter(s.charAt(first)))  first--; 
        return second - first; 
    }
}